com.blueokiris.bgrm.json

```diff
@@ -0,0 +1,70 @@
+{
+    "app-id": "com.blueokiris.bgrm",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "bgrm-fp.sh",
+    "finish-args": [
+        "--device=all",
+        "--socket=wayland",
+        "--socket=x11",
+        "--filesystem=host:ro"
+    ], "modules": [
+        {
+            "name": "virtualenv",
+            "buildsystem": "simple",
+            "build-options": {
+                "build-args": [
+                    "--share=network"
+                ]
```

Comment:
this is NOT allowed. you need to list the dependencies by hand.

Reply:
Removed and replaced with individual python dependencies.